sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -230,6 +231,24 @@ export interface HealthcareEntity extends Entity {
     relatedEntities: Map<HealthcareEntity, string>;
 }
 
+// @public
+export interface HealthcareEntityRelation {
+    relationType: HealthcareEntityRelationType;
+    roles: HealthcareEntityRelationRole[];
+}
+
+// @public
+export interface HealthcareEntityRelationRole {
+    entity: HealthcareEntity;
+    role: HealthcareEntityRelationRoleType;
```

Comment:
I am sorry. It should be type `string` with named "name" instead of "role". It is discussed in the last meeting we had.

Reply:
Here is the proposed design => https://apiview.dev/Assemblies/Review/329cfc783b474d97891a5349e705feb4?diffRevisionId=94926e8ff2954c6b8da109e47e5cc417&doc=False&diffOnly=False#com.azure.ai.textanalytics.models.AnalyzeHealthcareEntitiesResult

Reply:
I agree with the name `name` instead of `role` but I do prefer to have the enum `HealthcareEntityRelationRoleType` instead of a `string` .

Reply:
Renamed `role` to `name`.